docs/source/release/v6.12.0/inelastic.rst

```diff
@@ -7,23 +7,16 @@ Inelastic Changes
 
 New Features
 ------------
-.. amalgamate:: Inelastic/New_features
+- The deprecated Calculate Paalman Pings tab on the :ref:`Inelastic Corrections interface <interface-inelastic-corrections>` has been removed after no user feedback from v6.10 and v6.11.
+- The `Inelastic Data Processor <interface-inelastic-data-processor>` interface :math:`S(Q, \omega)` tab can now load ILL data.
```

Comment:
```suggestion
- The :ref:`Data Processor Interface <interface-inelastic-data-processor>` interface :math:`S(Q, \omega)` tab can now load ILL data.
```

Reply:
Good spot. I've edited your suggestion to remove the duplicate `interface`.